### PR TITLE
Init ETA by data-attribute with JSON format

### PR DESCRIPTION
**Pull request by vuquangpham**
We can init ETA with options by using JSON format in container attribute:

```html
    <div data-eta='{ "id":"my-id", "animation":"slide", "hash":"false", "duration":100}'>
        <!-- section 1 -->
        <div>
            <button data-eta-trigger="section-1">Section 1</button>
            <div data-eta-receiver="section-1">Content</div>
        </div>

        <!-- section 2 -->
        <div>
            <button data-eta-trigger="section-2">Section 2</button>
            <div data-eta-receiver="section-2">Content</div>
        </div>
    </div>
```

### CODE REVIEW

**Review by phucbm — CHANGES_REQUESTED** [review]
Set ID via JSON seems not to work. Please check.

File: src/helpers.js [hunk 1]
```diff
@@ -217,7 +217,6 @@ export function getOptions(context){
     // get ID
     if(options['id'] && !isEmptyString(options['id'])){
         context.id = options['id'];
-        delete options['id'];
```
Comment (phucbm) [inline src/helpers.js]: Who put this line here at the beginning? Looks kind of a new thing to me.

**Review by phucbm — COMMENTED** [review]
Just a minor question left.

**Review by phucbm — APPROVED** [review]
LGTM.

**Comment by phucbm** [comment]
Need update for demo and README.

**Comment by vuquangpham** [comment]
> Need update for demo and README.

I have updated it, please check again! Thanks.

**Review by phucbm — APPROVED** [review]
LGTM.

**Pull request merged**